src/main/java/it/unitn/disi/smatch/CLI.java

```diff
@@ -49,10 +60,12 @@
             " offline <input> <output>                   read input file, preprocess it and write it into output file\n" +
             " online <source> <target> <output>          read source and target files, run matching and write the output file\n" +
             " filter <source> <target> <input> <output>  read source and target files, input mapping, run filtering and write the output mapping\n" +
+            " allsteps <source> <target> <output>        read source and target files, run all steps from 1 to 4 and write the output mapping\n" +            
             "\n" +
             " Options: \n" +
             " -config=file.xml                           read configuration from file.xml instead of default s-match.xml\n" +
-            "                                            use -Dkey=value to supply values to ${key} placeholders in the config file";
+            "                                            use -Dkey=value to supply values to ${key} placeholders in the config file\n" +
+            " -v                                         turn verbose mode on\n";    
```

Comment:
Verbose flag sort of duplicates the DEBUG and TRACE levels in logging. Yes, there is no logging here in the main (maybe it should be), but now there are two ways to increase messaging detail: verbose flag and logging level. They are not exactly equivalent, but then one should introduce verbosity level as a framework-wide thing to allow components to be verbose too. An alternative would be to keep logging level here as well and log extra info at DEBUG or TRACE. 

What do you think, which alternative is better?